modules/ecs/variables.tf

```diff
@@ -13,11 +13,6 @@ variable "aws_partition" {
   description = "The AWS partition the services are being run in."
 }
 
-variable "aws_dns_suffix" {
-  type        = string
-  description = "The AWS DNS suffix for the region."
-}
```

Comment:
Not needed anymore as per [this](https://github.com/spacelift-io/terraform-aws-iam-spacelift-selfhosted/pull/4) PR